### PR TITLE
OSDOCS-4315 added architectures to Compliance Operator Supported Profiles table

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.6+

Issue:
[OSDOCS-4315](https://issues.redhat.com//browse/OSDOCS-4315)

Link to docs preview:
[Compliance Operator Supported Profiles table](http://file.rdu.redhat.com/antaylor/osdocs-4315/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles)

### CODE REVIEW

File: modules/compliance-supported-profiles.adoc [hunk 1]
```diff
@@ -8,86 +8,106 @@
...
-|rhcos4-moderate
-|NIST 800-53 Moderate-Impact Baseline for Red Hat Enterprise Linux CoreOS
-|0.1.39+
-|link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
-
-|rhcos4-nerc-cip
-|North American Electric Reliability Corporation (NERC) Critical Infrastructure Protection (CIP) cybersecurity standards profile for Red Hat Enterprise Linux CoreOS
-|0.1.44+
-|link:https://www.nerc.com/pa/Stand/Pages/CIPStandards.aspx[NERC CIP Standards]
+|`x86_64`
```
Comment (prb112) [inline modules/compliance-supported-profiles.adoc:94]: With the latest release of the compliance operator with 4.12, the ppc64le architecture supports PCI

Comment (rhmdnd) [reply modules/compliance-supported-profiles.adoc:94]: Latest upstream release (0.1.56)? Or downstream 0.1.53?

Comment (prb112) [reply modules/compliance-supported-profiles.adoc:94]: 0.1.56 on the upstream.

Comment (rhmdnd) [reply modules/compliance-supported-profiles.adoc:94]: Excellent - thanks for clarifying!

Comment (sheriff-rh) [reply modules/compliance-supported-profiles.adoc:94]: Thanks @prb112 , I can include that when version 0.1.56 is released downstream.

Comment (rhmdnd) [reply modules/compliance-supported-profiles.adoc:94]: We're working on landing https://github.com/ComplianceAsCode/compliance-operator/pull/153 and we'll be releasing 0.1.57 upstream.

**Review by rhmdnd — APPROVED** [review]
/lgtm

File: modules/compliance-supported-profiles.adoc [hunk 2]
```diff
@@ -8,86 +8,106 @@
...
-|Compliance Operator version 
+|Compliance Operator version
 |Industry compliance benchmark
+|Supported architectures
 
 |ocp4-cis
 |CIS Red Hat OpenShift Container Platform 4 Benchmark
 |0.1.39+
 |link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] footnote:cisbenchmark[To locate the CIS RedHat OpenShift Container Platform v4 Benchmark, go to  link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks] and type `Kubernetes` in the search box. Click on *Kubernetes* and then *Download Latest CIS Benchmark*, where you can then register to download the benchmark.]
+|`x86_64`
```
Comment (pneedle-rh) [inline modules/compliance-supported-profiles.adoc:24]: You could potentially remove the back-ticks from all of the supported architecture references in this column. My understanding is that they are short-hand architecture names and not direct code references, in this context.

**Review by pneedle-rh — COMMENTED** [review]
@sheriff-rh I have completed my review of this PR. Aside from one minor suggestion, the PR looks good to me!

**Comment by pneedle-rh** [comment]
I'm not sure if I should add the `enterprise-4.7` label or not and am currently looking into this.

**Comment by sheriff-rh** [comment]
@pneedle-rh I tried different formats including an unordered list but it never rendered correctly in the build preview. The architectures would just blend together on one line or wrap unevenly. So I added the backticks back to the lines. We have referenced the architectures in backticks elsewhere in docs, as well. 

As far as  4.7 is concerned, this is a quality of life update rather than any sort of feature added. I see no issue in cherrypicking to 4.7.

If you're okay with it, it's ready to merge, please. :rocket:

**Pull request merged**